config.go

```diff
@@ -27,7 +28,10 @@ var (
 
 // parseFlags sets up the flags and parses them, this needs to be called before any other operation
 func parseFlags() {
-	flag.String("localstatedir", "", "Directory to store local state information. Defaults to current working directory.")
+	flag.String("workdir", "", "Working directory for GlusterD. (default: current directory)")
```

Comment:
`workdir` is the default directory for the process. It will be the directory from which relative paths are rooted. Setting a `workdir` helps avoid clutter in the current directory caused by (possible) artifacts that get generated.
For eg. the way we launch etcd right now, we don't set a `data-dir` for etcd. etcd creates it's data-dir in its working directory. As a child of GD2, etcds working directory is GD2 working directory. This leads to a `default.etcd` directory being created in whichever directory you start GD2 from.

`localstatedir` is the place where information managed and generated by GD2 is stored. This is the `/var/lib/glusterd` directory.